fendermint/actors/eam/src/state.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::Cid;
+use fil_actors_runtime::{ActorError, Map2, DEFAULT_HAMT_CONFIG};
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::address::Address;
+
+pub type DeployerMap<BS> = Map2<BS, Address, ()>;
+
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone)]
+pub struct State {
+    /// The allowed deployers of contract
+    pub deployers: Cid, // HAMT[Address]()
+}
+
+impl State {
+    pub fn new<BS: Blockstore>(store: &BS) -> Result<State, ActorError> {
+        let empty_deployers = DeployerMap::empty(store, DEFAULT_HAMT_CONFIG, "empty").flush()?;
```

Comment:
What is this `"empty"` thing?

Reply:
It's an identifier for the `map`, it's not stored in the storage, but just in memory. I took this from the actor development convention. I think this is a identifier for the map or sth along that line.